api/src/main/resources/public/api/v1/schema/jobspec.schema.json

```diff
@@ -250,8 +250,7 @@
               "containerPath": {
                 "type": "string",
                 "description": "The path of the volume in the container",
-                "minLength": 1,
-                "pattern": "^/[^/].*$"
+                "minLength": 1
```

Comment:
v0 did NOT require this and we can NOT require it for v1.   The pattern is not allowed for FBS.

Reply:
Oh, yeah, relative paths are totes supposed to be supported, because they go in the Mesos sandbox, right?